app/controllers/cars_controller.rb

```diff
@@ -1,13 +1,19 @@
 class CarsController < ApplicationController
     # skip_before_action :verify_authenticity_token
+    # CODE_COMMENT: I see that most of the controller / view code you have written is in cars any reason for this? 
+    # Probabaly you could have also used dashboard controller well?
+    # CODE_COMMENT: Why is the above line there?
     before_action :authenticate_user!, except: [:index, :show, :allcars]
     before_action :correct_user, only: [:edit, :update, :destroy, :dashboard]
 
+    # CODE_COMMENT: Are we sure we want all the cars for the next 3 statements? Isnt that an overkil? We should do some sort of user related car fetching 
+    # or something on those lines correct?
+    #Is there any imp attributes that a car should have mandatorily. What happens if there are 1000 cars or more ?.
```

Comment:
def allcars
    @cars = Car.all.limit(10)
end

I've updated it this way so that, if there are many cars, it'll give only 10 at a time.
I also came across another gem 
https://rubygems.org/gems/will_paginate
Using this we can create many pages of 10 records at a time.